### PR TITLE
Music_album and genre classes

### PR DESCRIPTION
**Pull request by ezraermy**
In this pull request, I:
- Created music_album class
- Created genre class
- Create a unit test for all classes.

### CODE REVIEW

**Review by Muhammad0602 — APPROVED** [review]
> ## Status: Approved ✔️
## Hi @ezraermy 
> 
>Your project looks great; you have done it exceptionally well. 
> 
> ### Congratulations! 🎉 🎉 🎉 🎉
>
ApplaudingYesGIF
 
> 
> You may close this Pull Request and merge the changes Keep being awesome devs and good luck with your next milestone!
> 
> ### To Highlight
> ✅ Descriptive pull request message
   ✅ linter error checks validated 
   ✅ Documented your work professionally 
> 
> ### Optional suggestions
> Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

> Cheers and Happy coding! 👏 👏 👏

File: spec/music_album_spec.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+require 'date'
+require_relative '../classes/item/music_album.rb'
+
+describe MusicAlbum do
+  let(:album) { MusicAlbum.new('2010-01-01', true) }
```
Comment (Swarzstein) [inline spec/music_album_spec.rb:5]: Please initialize the MausicAlbum with a date as MusicAlbum.new(Date.new(2010, 1, 1), true) so publish_date can be appropriately recognized as a date object.

File: spec/music_album_spec.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+  it 'is an instance of MusicAlbum' do
+    expect(album).to be_an_instance_of MusicAlbum
+  end
+
+  it 'has a publish date' do
+    expect(album.publish_date).to eq '2010-01-01'
```
Comment (Swarzstein) [inline spec/music_album_spec.rb:12]: you can change here the date string to a date object.

File: spec/music_album_spec.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  end
+
+  it 'has a Spotify value' do
+    expect(album.on_spotify).to eq true
+  end
```
Comment (Swarzstein) [inline spec/music_album_spec.rb:17]: Please consider adding a test for the move_to_archive method or the can_be_archived? method.

**Review by Swarzstein — CHANGES_REQUESTED** [review]
Good job so far!
It's looking nice 👌🏼 


There are just some things you have to make some changes which are indicated below.

**Review by Swarzstein — APPROVED** [review]
Great job! Now you can Merge it!

**Pull request merged**